msteams-platform/toolkit/Teams-Toolkit-CLI.md

```diff
@@ -1,23 +1,516 @@
+* Manage multiple environments, Microsoft entra-apps and Teams app registrations.
+
+## Get started
+
+Install `teamsapp` from `npm` and run `teamsapp -h` to check all available commands:
+
+```bash
+  npm install -g @microsoft/teamsapp-cli
+  teamsapp -h
+```
+
+:::image type="content" source="../assets/images/teams-toolkit-cli/npm-package-install.png" alt-text="Screenshot shows the installation of npm package.":::
+
+## Supported commands
+
+| Command | Description |
+|----------------|-------------|
+| `teamsapp doctor` | Prerequisites to create Teams application.|
+| `teamsapp new`| Create new Teams application.|
+| `teamsapp add`| Adds features to your Teams application.|
+| `teamsapp auth`| Manage cloud service accounts. The supported cloud services are Azure and Microsoft 365. |
+| `teamsapp entra-app` | Manage the Microsoft Entra app in the current application. |
+| `teamsapp env` | Manage environments. |
+| `teamsapp help` |  Show Microsoft Teams Toolkit CLI help. |
+| `teamsapp install` | Upload a custom app for a given application package across Microsoft 365. |
+| `teamsapp launchinfo` | Get launch information of an acquired Microsoft 365 App. |
+| `teamsapp list` | List available Microsoft Teams application templates and samples. |
+| `teamsapp provision` | Provision cloud resources in the current application.|
+| `teamsapp deploy` | Deploy the current application. |
+| `teamsapp package` | Build your Teams app into package for publishing.|
+| `teamsapp validate` | Validate the current application.|
+| `teamsapp publish` | Publish the app to Teams.|
+| `teamsapp preview` | Preview the current application. |
+| `teamsapp update` |  Update the Teams app manifest to Teams Developer Portal. |
+| `teamsapp upgrade` | Upgrade the project to work with the latest version of Teams Toolkit. |
+| `teamsapp collaborator`| Collaborate with other developers in same project.|
+| `teamsapp uninstall` | Remove an acquired Microsoft 365 App. |
+
+:::image type="content" source="../assets/images/teams-toolkit-cli/list-of-commands.png" alt-text="Screenshot shows the list of available commands.":::
+
+## Global Options
+
+| Command | Description |
+|----------------|-------------|
+| `--version -v` | Display Microsoft Teams Toolkit CLI version.|
+| `--help -h` | Show Microsoft Teams Toolkit CLI help.|
+| `--interactive -i` | Run the command in interactive mode. The default value is true.|
+| `--debug` | Print debug information. The default value is false.|
+| `--verbose` | Print diagnostic information. The default value is false.|
+| `--telemetry` | Whether to enable telemetry. The default value is true.|
+
+:::image type="content" source="../assets/images/teams-toolkit-cli/global-options.png" alt-text="Screenshot shows the list of global options.":::
+
+## Interactive mode
+
+`teamsapp` command runs in interactive mode by default. You can work in non-interactive mode by setting `--interactive` flag to `false`.
```

Comment:
Different command has different default interactive mode, some commands are interactive by default, while others are non-interactive by default. The default value is specified in "-h" for each command.
<img width="883" alt="image" src="https://github.com/MicrosoftDocs/msteams-docs/assets/1658418/057d2227-3647-4b43-b1e5-302dcb5e8c44">
<img width="737" alt="image" src="https://github.com/MicrosoftDocs/msteams-docs/assets/1658418/edd2f638-266c-4408-91ca-50339472bef4">